jaxley/channels/non_capacitive/izhikevich.py

```diff
@@ -0,0 +1,58 @@
+# This file is part of Jaxley, a differentiable neuroscience simulator. Jaxley is
+# licensed under the Apache License Version 2.0, see <https://www.apache.org/licenses/>
+
+from typing import Optional
+from warnings import warn
+
+import jax
+import jax.numpy as jnp
+
+from jaxley.channels import Channel
+from jaxley.solver_gate import exponential_euler
+
+
+class Izhikevich(Channel):
+    """Izhikevich neuron model."""
+
+    def __init__(self, name: Optional[str] = None):
+        self.current_is_in_mA_per_cm2 = True
+        super().__init__(name)
+        self.channel_params = {
+            f"{self.name}_a": 0.02,
+            f"{self.name}_b": 0.2,
+            f"{self.name}_c": -65.0,
+            f"{self.name}_d": 8,
+        }
+        self.channel_states = {f"{self.name}_u": 0.0}
+        self.current_name = f"{self.name}_izhikevich"
+        warn(
```

Comment:
We could also raise `NotImplementetError` using [jax.custom_gradient](https://docs.jax.dev/en/latest/_autosummary/jax.custom_gradient.html) and replace this later with some sort of surrogate. Same goes for the fire model.